libraries/AC_WPNav/AC_Circle.cpp

```diff
@@ -23,6 +23,13 @@ const AP_Param::GroupInfo AC_Circle::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("RATE",    1, AC_Circle, _rate,    AC_CIRCLE_RATE_DEFAULT),
 
+    // @Param: RADUSED
+    // @DisplayName: Circle Radius used
+    // @Description: Use the radius at the center of the circle.
```

Comment:
Please improve this description

Reply:
@amilcarlucas san... Comments thanks. 
Change item name and description.